### PR TITLE
Added SoftShrink transfer operator for Keras

### PR DESCRIPTION
**Pull request by amitsrivastava78**
New operator along with test cases added to Keras

### CODE REVIEW

**Comment by amitsrivastava78** [comment]
@tanzhenyu , @pavithrasv , @nataliaponomareva & @rthadur can you pls review the PR

**Comment by amitsrivastava78** [comment]
@tanzhenyu , @pavithrasv , @nataliaponomareva & @rthadur can you pls review the PR

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @pavithrasv: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 14 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @pavithrasv: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 29 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @pavithrasv: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 14 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @pavithrasv: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 29 days with no activity and the `awaiting review` label has been applied.

**Comment by tensorflowbutler** [comment]
Nagging Reviewer @pavithrasv: You have been added as a reviewer to this pull request. Please add your review or reassign. It has been 44 days with no activity and the `awaiting review` label has been applied.

**Comment by pavithrasv** [comment]
This looks like a candidate for add-ons repo and we can may be move it to core if the layer is getting used widely.
